README.md

```diff
@@ -26,59 +26,86 @@ To install from source, follow the instructions in the [documentation](https://q
 
-Now that Qiskit is installed, it's time to begin working with Qiskit. To do this
-we create a `QuantumCircuit` object to define a basic quantum program.
+Now that Qiskit is installed, it's time to begin working with Qiskit. The essential parts of a quantum program are 
+1. Define and build a quantum circuit that represents the quantum state
+2. Define the classical output by a measurements circuit or a set of observable operators
+3. Depending on the output use the primitive function `sampler` to sample outcomes or the `estimaor` to estimate values.
+
+Usign the `QuantumCircuit` object a example quantum circuit can be created using:
 
 ```python
+import numpy as np
 from qiskit import QuantumCircuit
-qc = QuantumCircuit(2, 2)
-qc.h(0)
-qc.cx(0, 1)
-qc.measure([0,1], [0,1])
+
+# A quantum circuit for preparing the quantum state |000> + i |111>
+qc_example = QuantumCircuit(3)
+qc_example.h(0) # generate superpostion
+qc_example.p(np.pi/2,0) # add quantum phase
+qc_example.cx(0,1) # condition 1st qubit on 0th qubit
+qc_example.cx(0,2) # condition 2st qubit on 0th qubit
 ```
 
-This simple example makes an entangled state, also called a [Bell state](https://qiskit.org/textbook/ch-gates/multiple-qubits-entangled-states.html#3.2-Entangled-States-).
+This simple example makes an entangled state known as a GHZ state `(|000> + i |111>)/rt(2)`. It uses the standard quantum 
+gates: Hadamard gate, Phase gate and CNOT. 
 
-Once you've made your first quantum circuit, you can then simulate it.
-To do this, first we need to compile your circuit for the target backend we're going to run
-on. In this case we are leveraging the built-in `BasicAer` simulator. However, this
-simulator is primarily for testing and is limited in performance and functionality (as the name
-implies). You should consider more sophisticated simulators, such as [`qiskit-aer`](https://github.com/Qiskit/qiskit-aer/),
-for any real simulation work.
+Once you've made your first quantum circuit, you need to decide on which primtive function you will use. Starting with the sampler 
+we use the `compose` funtion to add a measurement circuit to the example circuit. In this example we simply map the qubits to
+the classical registers in asending order. 
 
 ```python
-from qiskit import transpile
-from qiskit.providers.basicaer import QasmSimulatorPy
-backend_sim = QasmSimulatorPy()
-transpiled_qc = transpile(qc, backend_sim)
+qc_measure = QuantumCircuit(3,3)
+qc_measure.measure([0,1,2], [0,1,2])
+qc_compose = qc_example.compose(qc_measure)
+
+from qiskit.primitives.sampler import Sampler
+sampler = Sampler()
+job = sampler.run(qc_compose, shots=1000)
+result = job.result()
+print(f" > Quasi probability distribution: {result.quasi_dists}")
 ```
-
-After compiling the circuit we can then run this on the ``backend`` object with:
+Running this will give the outcome `{0: 0.497, 7: 0.503}` which is `000` 50% of the time and `111` 50% of the time upto statistical errors.  
```

Comment:
The outcome is subject to statistical fluctuations. So, I think better would be:
```suggestion
Running this will give an outcome similar to `{0: 0.497, 7: 0.503}` which is `000` 50% of the time and `111` 50% of the time up to statistical fluctuations.  
```